assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -42,5 +42,18 @@ export const tests: Tests = {
     independent: true,
     seed: 7,
   },
+
+  upsellRecurringContributions: {
+    variants: ['control', 'benefitsOfBoth', 'shorterControl'],
+    audiences: {
+      US: {
+        offset: 0,
+        size: 1,
+      },
+    },
+    isActive: true,
+    independent: true,
+    seed: 0,
```

Comment:
Can you make this seed a different number from the other tests in this file? This will ensure independent segmentation from the other tests running on the site

Reply:
Is this necessary? - the documentation says the seed is only used in the case where `independent: false`

Reply:
Yes, you are right, my bad!

Reply:
It looks like the documentation could be wrong - @svillafe just pointed out that the code to determine the segments would indicate that the seed is only used when `independence` is `true`

```
function randomNumber(mvtId: number, independent: boolean, seed: number): number {
  if (!independent) {
    return mvtId;
  }

  const rng = seedrandom(mvtId + seed);
  return Math.abs(rng.int32());
}
```

Reply:
Here is a PR to change the docs: https://github.com/guardian/support-frontend/pull/500